packages/blockchain/src/genesisStates/index.ts

```diff
@@ -36,7 +36,7 @@ export async function genesisStateRoot(genesisState: GenesisState) {
         account.codeHash = Buffer.from(keccak256(toBuffer(code)))
       }
       if (storage !== undefined) {
-        const storageTrie = new Trie()
+        const storageTrie = new Trie({ useKeyHashing: true })
```

Comment:
Oh. The v6 final release clean-up bugs keep trickling in (still on a very modest level though).

Great find! 🙂

Reply:
yea it took quite a digging to find this one :sweat_smile:

Reply:
Just following up the call-path here,so this is actually not such a "mild" bug being triggered directly in `Blockchain.create()` in certain circumstances, right? 🤔 "Only" in genesis storage scenarios, but anyhow. Guess some bugfix releases are really getting closer to be realized/done at some point.

Reply:
yes in genesis storage scenarios only! thats why i digged around and used kiln genesis as well as kiln genesis block (finally had to find an old geth build to get it :sweat_smile: )